openoffice/index.html

```diff
@@ -61,265 +70,456 @@ <h1 align="center" class="content-header">Open Office</h1>
                         https://go.illinoiswcs.org/wcs-office
                     </a> -->
                 </p>
-                <p id="covid-message"> * = Officer </p>
+                <p id="week-indication"> </p>
+            </div> 
+        </div>
+        <div class="row padded" id="officehours">
+            <div class="tabs-left custom-tabs col-sm-offset-1 col-md-10">
+                <ul class="nav nav-tabs">
+                    <li id="week1tab"><a href="#week1" data-toggle="tab">Week 1</a></li>
+                    <li id="week2tab"><a href="#week2" data-toggle="tab">Week 2</a></li>
+                </ul>
+                <div class="tab-content col-md-10">
+                    <div class="tab-pane" id="week1">
+                        <table class="ui very basic collapsing celled table">
+                            
+                            <thead>
+                            <tr>
+                                <th style ="width: 10%;"> </th>
+                                <th class="openofficeheader"> Monday</th>
```

Comment:
separate words in class & id names using dashes (so this becomes `open-office-header`)